internal/graph/check_test.go

```diff
@@ -2998,3 +3005,531 @@ func TestIteratorReadStartingFromUser(t *testing.T) {
 		})
 	}
 }
+
+func TestConsumeUsersets(t *testing.T) {
+	type dsResults struct {
+		tuples []*openfgav1.TupleKey
```

Comment:
```suggestion
		tuples []*openfgav1.Tuple
```

makes the code easier to read down below in the test itself